translations/extensions/io_xls.py

```diff
@@ -0,0 +1,209 @@
+import datetime
+import glob
+import shutil
+import sys
+import zipfile
+import pandas as pd
+
+from openpyxl import Workbook
+from translations.translator import TranslationRequestGenerator, TranslationResponseProcessor
+from translations.utils import ConfigUtilities, Utilities
+
+
+class Constants:
+    DEFAULT_TRANSL_XLS_PATH = 'translations-xls/'
```

Comment:
Small nit: Can we standardize all these to either have a trailing `/` or not. Leads to a consistent way of referring to these directories and files throughout the code instead of some code adding a `+ '/'` and some not